config.toml

```diff
@@ -96,6 +96,14 @@
             static_theshold_value = 0.5
 
 
+    [DEFAULT.logging]
+        # Set up for logging
+        logger_name = "coref-model"
+        log_file = "coref_model_logs.log"
```

Comment:
Could we go for a more strict path for the log file without being dependent on `pwd`? For example, in the config initialization phase, we could specify the target directory

```python
dd.resolve().parent.joinpath(log_file)
# log_file, e.g., could always be in "data/logs/{experiment_name}_{timestamp}.log"
# with exp_name and timestamp being there to differentiate between runs
```

What are your thoughts, @sahanmar?